### PR TITLE
Fix-Configuring Account Locking due to Failed Login Attempts issues -[Master]

### PR DESCRIPTION
**Pull request by asekawa**
Added extra step for Google Account configuration for the email sending [Less secure app access]

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 1]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+
+	1.	To request password entry from the users who are registered via Management Console, add the following configurations. 
+
+
+	 	```
+	  	[event.default_listener.identity_mgt]
+	  	priority= "50"
+	  	enable = false
+	  	event.default_listener.governance_identity_mgt]
+	    priority= "95"
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:27]: does this render properly?

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 2]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
-	[event.default_listener.identity_mgt]
-	priority= "50"
-	enable = false
-	[event.default_listener.governance_identity_mgt]
-	priority= "95"
-	enable = true
-	```
+1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
+
+	1.	To request password entry from the users who are registered via Management Console, add the following configurations.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:19]:
```suggestion
	1.	To request password entry from the users who are registered via the management console, add the following configurations:
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 3]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+	 	```
+	  	[event.default_listener.identity_mgt]
+	  	priority= "50"
+	  	enable = false
+	  	event.default_listener.governance_identity_mgt]
+	    priority= "95"
+	    enable = true
+	    ```
+
+	2.	To configure the email server to send emails requesting password entry, add the following configurations.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:31]:
```suggestion
	2.	To configure the email server to send emails requesting password entry, add the following configurations:
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 4]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+
+		-	**from_address**: This is the email address from which the confirmation email will be sent.
+		-	**username**: This is the user name of the given email address.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:34]:
```suggestion
		-	**username**: This is the username of the given email address.
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 5]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+		password= "Wso2@iam70"
+		hostname= "smtp.gmail.com"
+		port= 587
+		enable_start_tls= true
+		enable_authentication= true
+		```
+
+		!!! warning "If you are using a Google email account"
+
+			Google has restricted third-party applications and less secure applications from sending emails by default. As WSO2 Identity Server acts as a third-party application when sending emails for password entry, follow the steps below to enable your Google email account to provide access to third-party applications.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:61]:
```suggestion
			Google has restricted third-party applications and less-secure applications from sending emails by default. As WSO2 Identity Server acts as a third-party application when sending emails for password entry, follow the steps below to enable your Google email account to provide access to third-party applications.
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 6]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+
+			1.	Access [https://myaccount.google.com/security](https://myaccount.google.com/security).
+
+			2.	Under **Signing in to Google** section, turn off the **2-step Verification** option.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:65]:
```suggestion
			2.	In the **Signing in to Google** section, turn off the **2-step Verification** option.
```

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 7]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
+
+			3.  Enable **Less secure app access** in Google Account security section.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:67]:
```suggestion
			3.  Enable **Less secure app access** in the Google Account's security section.
```

Comment (asekawa) [reply en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md]: commit b7eadef1dc48ae9349338ad98ab4e12a1e9d1319

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md (hunk 6 above)
Comment (asekawa) [reply en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md]: commit b7eadef1dc48ae9349338ad98ab4e12a1e9d1319

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md (hunk 2 above)
Comment (asekawa) [reply en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md]: commit b7eadef1dc48ae9349338ad98ab4e12a1e9d1319

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md (hunk 1 above)
Comment (asekawa) [reply en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:27]: Render-display


**The section is rendered properly**

File: en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md [hunk 8]
```diff
@@ -14,16 +14,57 @@ Let's learn how Sam implements this!
...
-	[event.default_listener.identity_mgt]
-	priority= "50"
-	enable = false
-	[event.default_listener.governance_identity_mgt]
-	priority= "95"
-	enable = true
-	```
+1.	Open the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
+
+	1.	To request password entry from the users who are registered via the management console, add the following configurations.
```
Comment (nilminiwso2) [inline en/docs/learn/configuring-account-locking-due-to-failed-login-attempts.md:19]:
```suggestion
	1.	To request password entry from the users who are registered via the management console, add the following configurations:
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**